libs/ui/src/components/NeoButton/NeoButton.scss

```diff
@@ -30,7 +30,15 @@
     }
 
     &--k-accent {
-      @apply bg-k-primary text-black;
+      @apply bg-k-primary ;
+    
+      &:not(:hover):not(:disabled) {
+        @apply text-black #{!important};
+      }
```

Comment:
```suggestion
        @apply !text-black;
```

Reply:
getting this 

``````
[plugin:vite:css] [sass] expected ";".
   ╷
36 │         @apply !text-black;
   │                ^
   ╵
  libs/ui/src/components/NeoButton/NeoButton.scss 36:16  @import
  libs/ui/src/components/NeoButton/NeoButton.vue 2:9     root stylesheet

`````

![CleanShot 2024-02-05 at 10 16 44@2x](https://github.com/kodadot/nft-gallery/assets/44554284/0f008a0c-ed6d-43f5-a73d-0fdd0bb65b02)

Reply:
ouch, lgtm then 👍🏻